src/data/requirements/camo/_index.js

```diff
@@ -0,0 +1,32 @@
+import classic from "./classic"
+import cliffside from "./cliffside"
+import digital from "./digital"
+import dragon from "./dragon"
+import foliage from "./foliage"
+import fun from "./fun"
+import geometric from "./geometric"
+import reptile from "./reptile"
+import skulls from "./skulls"
+import solidColours from "./solidColours"
+import sprayPaint from "./sprayPaint"
+import stripes from "./stripes"
+import tiger from "./tiger"
+import woodland from "./woodland"
+
+export {
+    classic,
+    cliffside,
+    digital,
+    dragon,
+    foliage,
+    fun,
+    geometric,
+    reptile,
+    skulls,
+    solidColours,
+    sprayPaint,
+    stripes,
+    tiger,
+    woodland
+}
```

Comment:
These should maybe be ordered by their appearance in-game rather than alphabetical, but we can change that later! 😃

Reply:
The export when as an object won't hold the order, formatters will override this as object property ordering isn't a thing in code.
We can shift them into an array (which preserve order) when I get to the point that I'm making the camo requirement file like you did with "[orion\src\data\requirements.js](https://github.com/carlssonemil/orion/blob/main/src/data/requirements.js)"